### ISSUE TITLE
Left facing arrow logic is faulty

### ISSUE THREAD

**Issue by Lixquid** [adrai/flowchart.js#26]
When depicting control flowing right or downwards, the diagram renders fine. However, when you attempt to force a leftwards operation, the arrow comes out wonky. Screenshot:

![Example](http://i.imgur.com/j9v2WRw.png)

Code:

```
st=>start: Start:>http://www.google.com[blank]
e=>end:>http://www.google.com
op1=>operation: My Operation
sub1=>subroutine: My Subroutine
cond=>condition: Yes 
or No?:>http://www.google.com
io=>inputoutput: catch something...

st->op1->cond
cond(yes)->io->e
cond(no,left)->sub1(left)->op1
```

**Comment by amitguptagwl** [comment]
I'm facing the same issue.

**Comment by mlopezgva** [comment]
I thought it was my fault, but I see there's still work to do. I tried `flowchart.js` as part of HackMD.io, but right now is too buggy. Also, the generating code gets messy and it is hard to follow.

![image](https://user-images.githubusercontent.com/996070/47485283-19f72780-d83e-11e8-805c-88d6b047d333.png)
